iocore/eventsystem/P_UnixSocketManager.h

```diff
@@ -55,21 +55,52 @@ transient_error()
   return transient;
 }
 
+#if !HAVE_ACCEPT4
+int
+accept4(int sockfd, struct sockaddr *addr, socklen_t *addrlen, int flags)
+{
+  int fd, err;
+
+  do {
+    fd = accept(sockfd, addr, addrlen);
+    if (likely(fd >= 0))
+      break;
+  } while (transient_error());
+
+  if ((fd >= 0) && (flags & SOCK_CLOEXEC) && (safe_fcntl(fd, F_SETFD, FD_CLOEXEC) < 0)) {
+    err = errno;
+    close(fd);
+    errno = err;
+    return -1;
+  }
+
+  if ((fd >= 0) && (flags & SOCK_NONBLOCK) && (safe_nonblocking(fd) < 0)) {
+    err = errno;
+    close(fd);
+    errno = err;
+    return -1;
+  }
+
+  return fd;
+}
+#endif
```

Comment:
`#else` so the literal `accept4` isn't compiled in places where it doesn't exist?